packages/zowe-explorer-api/src/fs/FsWatcher.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { Disposable as VSDisposable, FileSystemWatcher, Uri, workspace } from "vscode";
+
+interface CoreWatchers {
```

Comment:
Will this interface be helpful to export for exenters?
That way they know which `filewatchers` are available as part of the ZoweFsWatcher.

No strong feelings either way since the intention of the `watchers` is for internal use only (i.e. private with no getters)

Reply:
This was added for stronger typing for the `watchers` variable in `ZoweFsWatcher`.

We can keep this internal since only `ZoweFsWatcher` uses it, but I've adjusted it to be a type.
Addressed in 0ca207b8